src/Mvc/Mvc.TagHelpers/src/Cache/DistributedCacheTagHelperService.cs

```diff
@@ -39,6 +39,13 @@ public class DistributedCacheTagHelperService : IDistributedCacheTagHelperServic
         private readonly ILogger _logger;
         private readonly ConcurrentDictionary<CacheTagKey, Task<IHtmlContent>> _workers;
 
+        /// <summary>
+        /// Creates a new <see cref="DistributedCacheTagHelperService"/>.
+        /// </summary>
+        /// <param name="storage">The <see cref="DistributedCacheTagHelper"/>'s cache storage.</param>
```

Comment:
> Parameter comments should include the type

Ehhh, disagree. It's not super valuable for a parameter doc to also have the type since you typically see the type name when creating/invoking something anyhow. As an example, the docs for `Task` go out of their way to not repeat the type name:

![image](https://user-images.githubusercontent.com/2008729/53510919-ecccb700-3a73-11e9-978c-0ba4708896ea.png)

As with StreamReader and many others:
![image](https://user-images.githubusercontent.com/2008729/53511153-914ef900-3a74-11e9-87d0-bbfe82db9a50.png)

That being said I'll do it here for consistency.